memo/js/package.json

```diff
@@ -1,70 +1,77 @@
 {
-  "name": "@solana/spl-memo",
-  "version": "0.2.0",
-  "description": "SPL Memo Program JS API",
-  "files": [
-    "lib",
-    "src",
-    "LICENSE",
-    "README.md"
-  ],
-  "main": "lib/cjs/index.js",
-  "module": "lib/esm/index.mjs",
-  "types": "lib/types/index.d.ts",
-  "type": "module",
-  "sideEffects": false,
-  "exports": {
-    "import": "./lib/esm/index.mjs",
-    "require": "./lib/cjs/index.js",
-    "types": "./lib/types/index.d.ts"
-  },
-  "scripts": {
-    "build": "yarn clean && tsc -p tsconfig.json && tsc-esm -p tsconfig.json && tsc -p tsconfig.cjs.json",
-    "postbuild": "echo '{\"type\":\"commonjs\"}' > lib/cjs/package.json && echo '{\"type\":\"module\"}' > lib/esm/package.json",
-    "clean": "rimraf ./lib",
-    "deploy": "yarn docs && gh-pages --dist docs --dest memo/ts --dotfiles",
-    "docs": "rimraf docs && NODE_OPTIONS=--max_old_space_size=4096 typedoc",
-    "lint": "eslint --max-warnings 0 . && npm run pretty",
-    "lint:fix": "eslint . --fix && npm run pretty:fix",
-    "pretty": "prettier --check '{,{src,test}/**/}*.{j,t}s'",
-    "pretty:fix": "prettier --write '{,{src,test}/**/}*.{j,t}s'",
-    "test": "npm run test:unit && npm run test:e2e",
-    "test:unit": "jest test/unit",
-    "test:e2e": "start-server-and-test 'solana-test-validator -r -q' http://localhost:8899/health 'jest test/e2e'"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/solana-labs/solana-program-library"
-  },
-  "publishConfig": {
-    "access": "public"
-  },
-  "author": "Solana Maintainers <maintainers@solana.com>",
-  "license": "Apache-2.0",
-  "devDependencies": {
-    "@types/chai": "^4.3.1",
-    "@types/jest": "^28.1.1",
-    "@types/node": "^17.0.42",
-    "@types/node-fetch": "^2.6.1",
-    "@types/prettier": "^2.6.3",
-    "@typescript-eslint/eslint-plugin": "^5.28.0",
-    "@typescript-eslint/parser": "^5.28.0",
-    "chai": "^4.3.6",
-    "eslint": "^8.17.0",
-    "eslint-config-prettier": "^8.5.0",
-    "jest": "^28.1.1",
-    "prettier": "^2.7.0",
-    "process": "^0.11.10",
-    "rimraf": "^3.0.2",
-    "start-server-and-test": "^1.14.0",
-    "ts-jest": "^28.0.5",
-    "ts-node": "^10.8.1",
-    "typedoc": "^0.23.10",
-    "typescript": "^4.7.3",
-    "typescript-esm": "^2.0.0"
-  },
-  "dependencies": {
-    "@solana/web3.js": "^1.41.0",
-    "buffer": "^6.0.3"
-  }
+    "name": "@solana/spl-memo",
+    "description": "SPL Memo Program JS API",
+    "version": "0.3.1",
```

Comment:
Not a big deal, but 0.3.0 should probably be next -- was this version copy-pasted from Token?

Reply:
Oh, you're right. This should be 0.2.1, this is a nonbreaking change.